lib/msal-react/src/MsalAuthentication.tsx

```diff
@@ -63,8 +69,8 @@ export function useMsalAuthentication(
 }
 
 export const MsalAuthentication: React.FunctionComponent<IMsalAuthenticationProps> = props => {
-    const { username, loginHandler, children } = props;
-    const { msal } = useMsalAuthentication({ username, loginHandler });
+    const { username, interactionType, authenticationRequest, children } = props;
+    const { msal } = useMsalAuthentication({ username, interactionType, authenticationRequest });
```

Comment:
```suggestion
export const MsalAuthentication: React.FunctionComponent<IMsalAuthenticationProps> = ({ username, interactionType, authenticationRequest, children }) => {
    const { msal } = useMsalAuthentication({ username, interactionType, authenticationRequest });
```

Reply:
destructuring in the params I feel looks cleaner

Reply:
thanks for the suggestion! updated!